### PR TITLE
Initial login page commit

### PR DESCRIPTION
**Pull request by smlisk0630**
- Created components folder and login page subfolder with index.js file
- Fixed index.css code to use TailwindCSS instead of default code

### CODE REVIEW

File: client/src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import './index.css';
```
Comment (malenchite) [inline client/src/components/Login/index.js:2]: This should be removed or an index.css file should be created

File: client/src/components/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                id="login-close"
+              >
+                <path d="M14.53 4.53l-1.06-1.06L9 7.94 4.53 3.47 3.47 4.53 7.94 9l-4.47 4.47 1.06 1.06L9 10.06l4.47 4.47 1.06-1.06L10.06 9z"></path>
+              </svg>
+            </section>
+            <section className="flex justify-center sm:flex sm:items-start">
+              <form action="#" method="POST">
+                <section className="mt-3 text-center sm:mt-0 sm:ml-4 sm:text-left">
+                  {/* block sets input label to display:block so it displays above the input box */}
+                  <label className="block mb-2 text-indigo-500" for="email">
```
Comment (malenchite) [inline client/src/components/Login/index.js:50]: The "for" property needs to be "htmlFor" in React. This also applies to line 63.

**Review by malenchite — CHANGES_REQUESTED** [review]
See inline comments

File: client/src/components/Login/index.js (hunk 1 above)
Comment (smlisk0630) [reply client/src/components/Login/index.js:2]: The index.css file is in the src folder.

File: client/src/components/Login/index.js (hunk 2 above)
Comment (smlisk0630) [reply client/src/components/Login/index.js]: Property updated.

**Review by malenchite — APPROVED** [review]

**Pull request merged**